dex/farm/src/lib.rs

```diff
@@ -82,7 +80,6 @@ pub trait Farm:
         self.owner().set(&self.blockchain().get_caller());
         self.reward_token_id().set(&reward_token_id);
         self.farming_token_id().set(&farming_token_id);
-        self.pair_contract_address().set(&pair_contract_address);
```

Comment:
Since this is not initialized anymore, would it make sense to also delete the field?

Reply:
Done, now it's only in the farm-with-lock contract.